schemas/sequencing_alignment.json

```diff
@@ -199,6 +200,13 @@
                   "type": "string",
                   "pattern": "^[a-zA-Z0-9\\-_:\\.]+$"
                 },
+                "read_group_id_in_bam": {
+                  "type": [
+                    "string",
+                    "null"
+                  ],
+                  "pattern": "^[a-zA-Z0-9\\-_:\\.']+$"
```

Comment:
do we want to allow more special characters? I personally think we don't need to if we don't have to.